### ISSUE TITLE
Hugo CSS minify breaks AMP validation

### ISSUE THREAD

**Issue by scriptstar** [gohugoio/hugo#7300]
# Description

Hugo is stripping the space after a parenthesis when doing minification. This happens when using the --minify option.

### Un-minified:

```
<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
```

### Minified:

```
<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end)0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end)0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end)0s 1 normal both;animation:-amp-start 8s steps(1,end)0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
```

The issue is here `steps(1,end)0s.`

## What version of Hugo are you using?

`Hugo Static Site Generator v0.69.2/extended darwin/amd64 BuildDate: unknown`

## Does the issue happen in the latest release?
Yes

**Comment by pamubay** [comment]
Duplicate Issues https://github.com/gohugoio/hugo/issues/6699
and it's already fixed since [v0.70.0](https://github.com/gohugoio/hugo/releases/tag/v0.70.0) release

**Comment by moorereason** [comment]
Thanks, @pamubay.
